config/routes.rb

```diff
@@ -83,6 +83,8 @@
     end
   end
 
+  post "/logs/filter", to: "case_logs#filter"
```

Comment:
Does this need it's own route? Could it not just be a `GET index` request with extra URL params? That way you can bookmark or share a filter too?

Reply:
Yeah, that makes sense. Lemme shift stuff around